docs/modules/ROOT/pages/api/testing.adoc

```diff
@@ -0,0 +1,298 @@
+= Testing
+
+:stark: https://docs.starknet.io/architecture-and-concepts/cryptography/stark-curve/[Stark]
+:secp256k1: https://github.com/starkware-libs/cairo/blob/main/corelib/src/starknet/secp256k1.cairo[Secp256k1]
+
+This module provides various helper functions for declaring, deploying,
+and testing Smart Contracts using `snforge` toolchain from Starknet Foundry.
+
+```cairo
+use openzeppelin_testing;
+```
+
+The module isn't part of the `openzeppelin` package and to be accessible has to
+be added as a separate dependency in `Scarb.toml`:
+
+```
+[dev-dependencies]
+openzeppelin_testing = { git = "https://github.com/OpenZeppelin/cairo-contracts.git", tag = "v0.15.0-rc.0" }
+```
+
+== Test Utilities
+
+[.contract]
+[[testutils-common]]
```

Comment:
```suggestion
[[testing-common]]
```
For consistency. This applies for the rest of the files as well.

Reply:
It makes sense, changed the module reference name